frontend/src/app/sessions/user-sessions-wrapper/create-sessions/create-readonly-session/create-readonly-session-dialog.component.html

```diff
@@ -56,11 +55,29 @@ <h2>Open read-only Session</h2>
         administrator or project lead.
       </div>
     }
+
+    @if (
+      maxNumberOfModels &&
+      this.selectedModelOptions().length > maxNumberOfModels
+    ) {
+      <div class="my-1 border p-2 text-sm shadow">
+        <p class="text-error">
+          The selected tool model only supports up to
+          {{ maxNumberOfModels }} session at a time.
+        </p>
+      </div>
+    }
+
     <div class="flex justify-between">
       <button mat-flat-button mat-dialog-close type="button">Cancel</button>
       <button
         mat-flat-button
-        [disabled]="this.selectedModelOptions().length === 0 || loading"
+        [disabled]="
+          this.selectedModelOptions().length === 0 ||
+          (maxNumberOfModels &&
+            this.selectedModelOptions().length > maxNumberOfModels) ||
```

Comment:
Since you use the condition twice (here and in the above if-statement), I'd outsource it to the ts file.

Reply:
Done moved it to a `maxNumberOfModelsExceeded` function.